### PR TITLE
 New section on using Azure native service broker

### PR DESCRIPTION
**Pull request by cjschroder**

### CODE REVIEW

File: xml/cap_depl_azure.xml [hunk 1]
```diff
@@ -634,7 +634,167 @@ kube:
...
+<para>
+    Test your new service broker with an example Rails app. First create a space
+    and org for your test app:
+</para>
+<screen>
+&prompt.user;cf create-org <replaceable>testorg</replaceable>
+
+&prompt.user;cf create-space <replaceable>scftest</replaceable> -o <replaceable>testorg</replaceable>
+
+yh8lgt_987654/ gprompt.user;cf target -o "<replaceable>testorg</replaceable>" -s "<replaceable>scftest</replaceable>"
```
Comment (btat) [inline xml/cap_depl_azure.xml]: Not sure what happened at the start there :D

File: xml/cap_depl_azure.xml [hunk 2]
```diff
@@ -634,7 +634,167 @@ kube:
...
+
+&prompt.user;cf create-service azure-mysql-5-7 basic <replaceable>scf-rails-example-db</replaceable>\
```
Comment (btat) [inline xml/cap_depl_azure.xml]: Space before the \ at the end needed

File: xml/cap_depl_azure.xml (hunk 1 above)
Comment (cjschroder) [reply xml/cap_depl_azure.xml]: The cat. Always it is the cat's fault.

**Comment by cjschroder** [comment]
Re https://github.com/SUSE/doc-cap/issues/117

**Pull request merged**